draft-ietf-ppm-dap.md

```diff
@@ -1586,82 +1581,91 @@ struct {
 } AggregationJobContinueReq;
 ~~~
 
-The `round` field is the round of VDAF preparation that the Leader just reached
+The `round` field is the round of DAP preparation that the Leader just reached
```

Comment:
In order to avoid conflating "VDAF rounds" with "DAP rounds" I think it would be helpful to rename `round`. How about `request_num`?

Reply:
(I renamed "DAP round" to "step" or "aggregation step", as "request number" led to text that was somewhat more awkward/confusing, but I definitely agree with the intent of the suggestion.)